src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -561,20 +561,20 @@
           BeforeTargets="Build"
           Condition=" '$(CleanWhileBuilding)' == 'true' ">
     <Message Text="DirSize Before Building $(RepositoryName)" Importance="High" />
-    <Exec Command="df --block-size=1M $(ProjectDir)" />
+    <Exec Command="df -m $(ProjectDir)" />
```

Comment:
Is `-m` a documented option?  I tried it in a few environments and it works but I don't see it documented.

Reply:
It appears not: https://github.com/coreutils/coreutils/blob/dd614d6011f64e06852adc532e7357de59f4bac3/src/df.c#L1662-L1665

Looks like it was removed from the documentation at the same time the long form `--megabytes` was deprecated (back in 2001), though only the long form was actually removed since that isn't needed for BSD compatibility.

An alternative if you want to avoid `-m` would be the `BLOCKSIZE` environment variable.

Reply:
Is `-h` a viable option?  I feel like this would also provide an acceptable end user experience here.  If not, I have no objection to setting the environment variable as suggested.

Reply:
Sounds good to me. I played it safe and went like-for-like but `-h` sounds like a good option.